.github/workflows/assets.yml

```diff
@@ -18,9 +18,11 @@ on:
       - Publish
     types:
       - completed
-    branches-ignore:
+    branches:
       - "[0-9]+.[0-9]+"
       - "[0-9]+.[0-9]+-stable"
+    tags:
```

Comment:
Based on the documentation I don't think tags filter is supported for workflow_run.

Thus let's not add it and see what happens.

If we want this I think we need to add code e.g., to get the sha of the head and check if that head is associated with a tag of this form, but if there are no ill effects to run it on all release branches then we don't need such logic.

Reply:
removed the tag filter.